### PR TITLE
feat(filter-field): Adds seperate disabled member in tags

### PR DESCRIPTION
**Pull request by rowa-audil**
### <strong>Pull Request</strong>

APM-283060

#### Type of PR

Feature (non-breaking change which adds functionality)

#### Checklist

- [ ] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by tomheller** [comment]
Depends on #1955 consider this one blocked until it is merged.

**Comment by ffriedl89** [comment]
I am really unsure about this - maybe we can get some eyes on this from UX. This seems to be really distracting - the more I try this the more I get annoyed by the tags constantly changing states. What about a little different solution - to not change any visuals but use a block cursor and maybe add a title attribute to the tags that states you are currently creating / editing another tag?

File: libs/barista-components/filter-field/src/filter-field-tag/filter-field-tag.scss
Comment (tomheller) [inline libs/barista-components/filter-field/src/filter-field-tag/filter-field-tag.scss:52]: could be $white for consistency, but is fine like that as well.

File: libs/barista-components/filter-field/src/filter-field-tag/filter-field-tag.scss
Comment (tomheller) [inline libs/barista-components/filter-field/src/filter-field-tag/filter-field-tag.scss:32]: why are we changing the color here?

File: libs/barista-components/filter-field/src/filter-field-tag/filter-field-tag.ts
Comment (tomheller) [inline libs/barista-components/filter-field/src/filter-field-tag/filter-field-tag.ts:100]:
```suggestion
  /** @internal Whether the tag is temporarily disabled or not. Based on editmode state */
```

**Review by tomheller — DISMISSED** [review]

**Review by tomheller — APPROVED** [review]

**Comment by tomheller** [comment]
Cherry picker failed to CP this into the minor branch, because the original feature is not yet in the minor. This is fine.

**Pull request merged**
